pom.xml

```diff
@@ -354,6 +354,13 @@
                         <exclude>src/main/java/com/alibaba/nacos/common/packagescan/classreading/**</exclude>
                         <exclude>.flattened-pom.xml</exclude>
                         <exclude>**/announcement.conf</exclude>
+                        <exclude>**/reflect-config.json</exclude>
```

Comment:
Why add ignored by rat but not add copyright?

Reply:
native-image-agent: Error: cannot parse filter file C:\Users\Darren\IdeaProjects\nacos\client\filter-config.json: com.oracle.svm.core.util.json.JSONParserException: line 1 column 0 Expected json literal but found /

If the json file add copyright it will not work

Reply:
OK